CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService jokeService;
+        private IJokeOutput jokeOutput;
+
+        public IJokeService JokeService { get => jokeService; private set => jokeService = value??throw new ArgumentNullException(); }
+        public IJokeOutput JokeOutput { get => jokeOutput; private set => jokeOutput = value??throw new ArgumentNullException(); }
+
+        public Jester(IJokeService jokeService, IJokeOutput jokeOutput)
+        {
+            if(jokeService == null){throw new ArgumentNullException();}
```

Comment:
Inline if statements do not need braces. Specify argument name.
```suggestion
            if (jokeService == null) throw new ArgumentNullException(nameof(jokeService));
```